trestle/core/commands/merge.py

```diff
@@ -146,37 +138,22 @@ def merge(cls, effective_cwd: Path, element_path: ElementPath, trestle_root: Pat
                 f'Target model not found. Possibly merge of the elements not allowed at this point. {str(e)}'
             )
         target_model_path = effective_cwd / destination_model_alias
-        logger.debug(
-            f'look for target model path {target_model_path} at dest alias {destination_model_alias} rel to cwd'
-        )
+
         # target_model filename - depends whether destination model is decomposed or not
-        if target_model_path.exists():
-            logger.debug(
-                f'target model path does exist so target path is subdir with target alias {target_model_alias}'
-            )
-            target_model_path = target_model_path / target_model_alias
-        else:
-            logger.debug(f'target model filename does not exist so target path is target alias {target_model_alias}')
-            target_model_path = target_model_path / target_model_alias  # FIXME this is same as above
-        logger.debug(f'final target model path is {target_model_path}')
+        target_model_path = target_model_path / target_model_alias  # FIXME this is same as above
 
         # if target model is a file then handle file. If file doesn't exist, handle the directory,
         # but in this case it's a list or a dict collection type
         target_model_filename = target_model_path.with_suffix(file_ext)
         if target_model_filename.exists():
-            logger.debug(f'target model path with extension does exist so load distrib {target_model_filename}')
             _, _, target_model_object = ModelUtils.load_distributed(target_model_filename, trestle_root)
         else:
             target_model_filename = Path(target_model_path)
-            logger.debug(f'target model path plus extension does not exist so load distrib {target_model_filename}')
-            logger.debug(f'get collection type for model type {target_model_type}')
```

Comment:
I think it's fine to remove these debug lines but split and merge are hard to debug and it may be helpful to convert some to trace calls rather than remove them.

Reply:
Moved to trace instead